internal/util/gitlab/gitlab.go

```diff
@@ -27,6 +27,8 @@ const (
 	CommitStatusSuccess CommitStatus = "success"
 	// CommitStatusFailure represents a failure of commit status
 	CommitStatusFailure CommitStatus = "failed"
+	// CommitStatusPending represents a failure of commit status
```

Comment:
Should be pending or not?
```suggestion
	// CommitStatusPending represents a pending of commit status
```